From bce7957106c332ef33b3629383f82db81e0a6f95 Mon Sep 17 00:00:00 2001
From: Umesh Agashe <uagashe@cloudera.com>
Date: Mon, 5 Feb 2018 14:53:29 -0800
Subject: [PATCH] HBASE-19915 (addendum): Fixed a typo because of which only
 daughterA was getting stored with a CLOSED state and not daughterB

---
 .../src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java b/hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java
index 5dc0565162c09b71258c8146a6f62448e17f47be..dad9aefcbc56408c6b2a26ac8534e0e2645f954b 100644
--- a/hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java
+++ b/hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java
@@ -1759,7 +1759,7 @@ public class MetaTableAccessor {
       // master tries to assign these offline regions. This is followed by re-assignments of the
       // daughter regions from resumed {@link SplitTableRegionProcedure}
       addRegionStateToPut(putA, RegionState.State.CLOSED);
-      addRegionStateToPut(putA, RegionState.State.CLOSED);
+      addRegionStateToPut(putB, RegionState.State.CLOSED);
 
       addSequenceNum(putA, 1, -1, splitA.getReplicaId()); //new regions, openSeqNum = 1 is fine.
       addSequenceNum(putB, 1, -1, splitB.getReplicaId());
-- 
2.16.1

